requirements.txt

```diff
@@ -2,6 +2,5 @@ pytest>=3.0.6
 numpy>=1.8
 avro-python3>=1.8
 shapely>=1.6b3
-rasterio>=0.36.0
```

Comment:
It appears that taking Rasterio out has caused Travis to fail: https://travis-ci.org/locationtech-labs/geopyspark/jobs/202749661#L4707

Are you installing it else where? Rasterio is not needed for any of the actual code to function, however, it's required for the tests.